HARK/ConsumptionSaving/ConsIndShockModelFast.py

```diff
@@ -1116,22 +1117,6 @@ def update_solution_terminal(self):
         )
 
     def post_solve(self):
-        """
-        Defines the value and marginal value functions for this period.
-        Uses the fact that for a perfect foresight CRRA utility problem,
-        if the MPC at :math:`t` is :math:`\\kappa_{t}`, and relative risk
-        aversion is :math:`\\rho`, then the inverse value function ``vFuncNvrs`` has a
-        constant slope of :math:`\\kappa_{t}^{-\\rho/(1-\\rho)}` and
-        ``vFuncNvrs`` has value of zero at the lower bound of market resources
-        `mNrmMin`.  See the `PerfForesightConsumerType <https://hark.readthedocs.io/en/latest/example_notebooks/PerfForesightConsumerType.html?highlight=PerfForesightConsumerType#Solution-method-for-PerfForesightConsumerType>`_ documentation notebook
-        for a brief explanation and the links below for a fuller treatment.
-
-        `PerfForesightCRRA/#vFuncAnalytical <https://www.econ2.jhu.edu/people/ccarroll/public/lecturenotes/consumption/PerfForesightCRRA/#vFuncAnalytical>`_
-
-        `SolvingMicroDSOPs/#vFuncPF <https://www.econ2.jhu.edu/people/ccarroll/SolvingMicroDSOPs/#vFuncPF>`_
-
-        """
-
```

Comment:
In the original file, this is the docstring of a method, which would be rendered to the Sphinx docs.

You seem to have moved it elsewhere in the function.

I gather that the description of the math applies better to the later code.

But are there no docstrings for methods in this class?

Reply:
I think the goal here was mainly to fix the urls, and if the docstring was moved so that it no longer applies to the class that is probably a mistake.  I have yet to devote any time to mastering how to use sphinx in docstrings.  

So, please just move it back to being the docstring.